experimental/packages/sdk-logs/src/export/SdkLogRecord.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { HrTime, SpanContext } from '@opentelemetry/api';
+import { InstrumentationScope } from '@opentelemetry/core';
+import type {
+  AnyValue,
+  LogBody,
+  LogAttributes,
+  SeverityNumber,
+} from '@opentelemetry/api-logs';
+import type { Resource } from '@opentelemetry/resources';
+
+/**
+ * A recording of a event. Typically the record includes a timestamp indicating when the
+ * event happened as well as other data that describes what happened, where it happened, etc.
+ *
+ * @remarks
+ * This interface is **not intended to be implemented by users**.
+ * To produce logs, use {@link Logger#emit}. To consume logs, implement {@link LogRecordProcessor#onEmit}.
+ * SdkLogRecord instances are created and managed by the SDK.
+ */
+export interface SdkLogRecord {
```

Comment:
Can we keep the same name "LogRecord"? that would prevent some issues with people already using LogRecord from this package